src/client/pages/ConsentsConfirmation.tsx

```diff
@@ -182,7 +186,7 @@ export const ConsentsConfirmation = ({
                   ))}
                 </ReviewTableRow>
               )}
-              {!optedOutOfProfiling && (
+              {!!optedIntoProfiling && (
```

Comment:
If this is a boolean value, you don't need the `!!`

```suggestion
              {optedIntoProfiling && (
```